docs/architecture/raft.rst

```diff
@@ -256,11 +256,12 @@ In summary, Raft makes schema changes safe, but it requires that a quorum of nod
 
 Handling Failures
 ------------------
-Raft requires a quorum of nodes in a cluster to be available. If one or more nodes are down, but the quorum is live, reads, writes,
-and schema updates proceed unaffected.
+Raft requires a quorum of nodes in a cluster to proceed. As long as the majority of nodes are alive and connected, schema and topology updates proceed unaffected.
 When the node that was down is up again, it first contacts the cluster to fetch the latest schema and then starts serving queries.
 
-The following examples show the recovery actions depending on the number of nodes and DCs in your cluster.
+ScyllaDB data path, read and write requests, works with :term:`Consistency Level (CL)` , and are not dependent on Raft quorum.
```

Comment:
```suggestion
ScyllaDB data path read and write requests work with :term:Consistency Level (CL) and is not dependent on Raft quorum.
```

Reply:
Either "work"+"are" (both plural, referring to "read and write requests"), or "works"+"is" (both singular, referring to "data path"). Here you combined "work" (plural) with "is" (singular) which doesn't make sense.